.github/workflows/cargo.yml

```diff
@@ -0,0 +1,89 @@
+name: Various cargo commands
+
+on:
+  push:
+    branches:
+      - main  # Set a branch name to trigger deployment
+  pull_request:
+
+jobs:
+  check:
```

Comment:
Unless I am missing something, looks like we can use `strategy` here. something like following?
```yaml
strategy:
	matrix:
		commands: [fmt, build, test, check]
```

Reply:
I haven't tried it yet. Can you add it?